### PR TITLE
libxaw3d: new formula

### PR DESCRIPTION
**Pull request by maxim-belkin**
### Contributing to homebrew-xorg:

_You can erase any parts of this template not applicable to your Pull Request._

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/homebrew-xorg/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/homebrew-xorg/pulls) for the same update/change?
- [x] Does your submission pass
`brew audit --strict <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

---
new libXaw3d formula

### CODE REVIEW

**Comment by maxim-belkin** [comment]
```
==> /home/linuxbrew/.linuxbrew/bin/gcc-5 test.c -o test
.
Last 150 lines from /home/ubuntu/.cache/Homebrew/Logs/libxaw3d/test.01.gcc-5:
2017-11-28 18:46:56 +0000

/home/linuxbrew/.linuxbrew/bin/gcc-5
test.c
-o
test

/home/linuxbrew/.linuxbrew/Cellar/gcc/5.4.0/libexec/gcc/x86_64-unknown-linux-gnu/5.4.0/cc1: error while loading shared libraries: libmpfr.so.4: cannot open shared object file: No such file or directory
Error: linuxbrew/xorg/libxaw3d: failed
Failed executing: /home/linuxbrew/.linuxbrew/bin/gcc-5 test.c -o test
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - run: git config --global user.name LinuxbrewTestBot
+      - run: git config --global user.email testbot@linuxbrew.sh
+      - run: chmod 0644 *.rb
+      - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
+      - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew/$CIRCLE_PROJECT_REPONAME
+      - run: mkdir /home/linuxbrew/bottles
+      - run: >-
+          cd /home/linuxbrew/bottles;
+          umask 022;
+          PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH";
```
Comment (sjackman) [inline .circleci/config.yml]: delete this blank line

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          brew install patchelf
+          && brew test-bot
+      - persist_to_workspace:
+          root: /home/linuxbrew
+          paths:
+            - bottles
```
Comment (sjackman) [inline .circleci/config.yml]: Delete lines 33–36.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      - store_test_results:
+          path: /home/linuxbrew/bottles
+  collect-bottles:
+    docker:
+      - image: linuxbrew/linuxbrew
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
```
Comment (sjackman) [inline .circleci/config.yml]: Delete lines 39–44.

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      - store_artifacts:
+          path: /tmp/workspace/bottles
+          destination: bottles
+notify:
+  webhooks:
+    - url: https://p4142ivuwk.execute-api.us-west-2.amazonaws.com/prod/ci-upload
+
```
Comment (sjackman) [inline .circleci/config.yml]: Delete this blank line.

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Merged. Thanks, Maxim!

**Pull request closed without merging**